app/components/course_preview/missing_information_component.rb

```diff
@@ -36,9 +40,14 @@ def train_with_disability_link = "#{about_publish_provider_recruitment_cycle_pat
     def train_with_us_link = "#{about_publish_provider_recruitment_cycle_path(provider_code, recruitment_cycle_year, course_code:, goto_preview: true)}#train-with-us"
 
     def about_accrediting_provider_link
-      return "#{edit_publish_provider_recruitment_cycle_accredited_provider_path(provider_code, recruitment_cycle_year, accredited_provider_code: @course.accredited_provider_code, goto_preview: true)}#accredited-provider-form-description-field" if FeatureService.enabled?(:accredited_provider_search)
-
-      "#{about_publish_provider_recruitment_cycle_path(provider_code, recruitment_cycle_year, course_code:, goto_preview: true)}#accrediting-provider-#{accrediting_provider.provider_code}"
+      if FeatureService.enabled?(:accredited_provider_search) && accrediting_provider_present?(course)
+        "#{edit_publish_provider_recruitment_cycle_accredited_provider_path(provider_code, recruitment_cycle_year, accredited_provider_code: @course.accredited_provider_code, goto_preview: true)}#accredited-provider-form-description-field"
+      elsif FeatureService.enabled?(:accredited_provider_search) && !accrediting_provider_present?(course)
+        publish_provider_recruitment_cycle_accredited_providers_path(provider_code,
```

Comment:
This link brings the user to a place they can add an accredited provider to the courses provider. What if it's the other way around? What if the course doesn't have the correct accrediting provider but the courses provider does?

Reply:
Good question. The intention of the link here is to demonstrate to the provider that they have not filled out information about the accredited provider. The link takes them to a place where they can add this information. Ideally, we would take them to the edit page, but because this is a new section, many providers haven't populated it yet and so the edit page doesn't exist. This is a work around.

When the provider adds the accredited provider in that joruney and inputs the "about accredited provider" info (which is mandatory). That info would then display on the preview page in place of the link.